src/index.tsx

```diff
@@ -79,6 +81,8 @@ function Index() {
                 { path: 'modules', element: <Modules /> },
                 { path: 'assignment', element: 'Actividades' },
                 { path: 'leaderboard', element: 'Leaderboard' },
+                { path: 'group', element: <Group /> },
+                { path: 'graph', element: <GroupGraphController /> },
```

Comment:
Change the grapth path to 'group/grap'. Otherwise the group tab in our class navbar wont be showing up as active.